extensions/azure-keyvault/runtime/src/main/java/io/quarkiverse/azure/keyvault/secret/runtime/config/KeyVaultSecretConfigUtil.java

```diff
@@ -0,0 +1,85 @@
+package io.quarkiverse.azure.keyvault.secret.runtime.config;
+
+import com.azure.security.keyvault.secrets.models.KeyVaultSecretIdentifier;
+
+public class KeyVaultSecretConfigUtil {
+    private static final String AZURE_KEYVAULT_PREFIX = "kv//";
+    private static final String AZURE_KEYVAULT_ENDPOINT_PREFIX = "https://";
+    private static final String AZURE_VAULT_URL_FORMAT = "https://%s.vault.azure.net/secrets/%s/%s";
```

Comment:
As discussed at standup, maybe AZURE_VAULT_URL_FORMAT can be derived from endpoint if provided, then fallback to the default one if not?

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/266/commits/2de155a260f02e9308881d704eb6f9199db9383a